web/src/components/PasswordWithMeter.js

```diff
@@ -49,10 +49,11 @@ class Password extends Component {
           <Choice
             className="password-input--show-password ds-u-float--right"
             checked={showPassword}
-            name="show password"
-            value="Show password"
+            name="show-password"
+            value="on"
             onChange={this.toggleShowPassword}
             size="small"
+            type="checkbox"
           >
             Show password
           </Choice>
```

Comment:
When upgrading from v2.0.0 of @cms/design-system, the label for this component will become a prop, instead of using the component's children as the label. https://github.com/CMSgov/design-system/releases/tag/core-2.0.2

Reply:
Currently, upgrading the design system from v2.0.0 causes this error:

```
ERROR in ./src/styles/index.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/resolve-url-loader!./node_modules/postcss-loader/src!./node_modules/sass-loader/dist/cjs.js??ref--5-4!./src/styles/index.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: File to import not found or unreadable: packages/required.
        on line 28 of node_modules/@cmsgov/design-system/dist/scss/settings/uswds/_index.scss
        from line 2 of node_modules/@cmsgov/design-system/dist/scss/settings/_index.scss
        from line 5 of node_modules/@cmsgov/design-system/dist/scss/index.scss
        from line 2 of /Users/rd/fearless/eapd/web/src/styles/index.scss
>> @import 'packages/required';
```

Reply:
Which, I just learned is fixed in v2.1.1